multicluster/charts/linkerd-multicluster/values.yaml

```diff
@@ -1,6 +1,16 @@
 gateway:
   # -- If the gateway component should be installed
   enabled: true
+  image:
+    # -- Docker registry for the gateway instance
+    registry: gcr.io
+    # -- Docker image name for the gateway instance
+    name: google_containers/pause
+    # -- Docker image tag for the gateway instance
+    tag: 3.2
+    # -- Pull policy for the gateway instance
+    # @default -- imagePullPolicy
+    pullPolicy: ""
```

Comment:
I think we should leave this out on this PR, as #9566 is tackling it separately

Reply:
I understand but I find it a pity that it creates a new way to declare the image. It makes the whole chart a bit less coherent.